Source/SwiftLintFramework/Extensions/Configuration+Parsing.swift

```diff
@@ -183,7 +183,7 @@ extension Configuration {
 
             switch rulesMode {
             case .allEnabled:
-                return
+                break
```

Comment:
I need to see if this is a rule that needs to be updated or a regression in SwiftSyntax.

Reply:
It's a regression: https://github.com/apple/swift-syntax/issues/851

I can work around it for now though.

Reply:
Fixed in https://github.com/apple/swift-syntax/pull/842